mlflow/server/js/src/utils/Utils.js

```diff
@@ -114,21 +114,32 @@ class Utils {
     return /[@/]github.com[:/]([^/.]+)\/([^/#]+)#?(.*)/;
   }
 
-  /**
-   * Renders the source name and entry point into an HTML element. Used for display.
-   * @param run MlflowMessages.RunInfo
-   * @param tags Object containing tag key value pairs.
-   */
+  static getBitBucketRegex() {
+    return /[@/]bitbucket.org[:/]([^/.]+)\/([^/#]+)#?(.*)/;
+  }
+
+    /**
+     * Renders the source name and entry point into an HTML element. Used for display.
+     * @param run MlflowMessages.RunInfo
+     * @param tags Object containing tag key value pairs.
+     */
   static renderSource(run, tags) {
     let res = Utils.formatSource(run);
     if (run.source_type === "PROJECT") {
       const match = run.source_name.match(Utils.getGitHubRegex());
+      const match1 = run.source_name.match(Utils.getBitBucketRegex());
```

Comment:
Give these slightly more readable names, such as githubMatch and bitBucketMatch.

Reply:
Done